packages/apps/plugins/plugin-map/src/components/MapControl.tsx

```diff
@@ -52,11 +52,9 @@ export const MapControl: FC<{ zoom?: number; center?: LatLngExpression; markers?
 
       {/* Markers. */}
       {/* TODO(burdon): Marker icon doesn't load on mobile. */}
-      {markers
-        .filter(({ location }) => location?.lat !== undefined && location?.lng !== undefined)
```

Comment:
Removed this filter since `lat` and `lng` should always be present.

Context from `@types/leaflet`:
```
export interface LatLngLiteral {
    lat: number;
    lng: number;
}
```